claro.theme

```diff
@@ -129,6 +138,10 @@ function claro_element_info_alter(&$type) {
   if (isset($type['button'])) {
     $type['button']['#attached']['library'][] = 'core/modernizr';
   }
+
+  if (!empty($theme_registry['admin_block_content'])) {
+    $theme_registry['admin_block_content']['variables']['attributes'] = [];
```

Comment:
What is this for?

Reply:
Wrong hook.

Reply:
Basically, I want to allow defining attributes for this theme function since I need to remove the default padding around an `admin_block_content` (admin task list) if it's the content of a panel component (`admin_block`).